Resources/config/bazinga_js_translation.yml

```diff
@@ -0,0 +1,28 @@
+active_domains:
+    - 'admin_ui_modules_mfu_drop_area_component'
```

Comment:
I think @lserwatka mentioned that recently that we want translation domains to be clear for the editors, not developers. I don't think using `component` and `module` is something editors would like to see. I also don't see the point in prefixing everything with `admin_ui`. Please refer to the existing domain names for best example.

Reply:
This is actually old stuff, @adamwojs has not generated new translations yet. I wated to get feedback about labels first. New labels are in code.
I should have made it clear, sorry for that.

Currently this label is: `mfu_drop_area_component`. I can delete `component` part.